src/lib/music/providers/ReplicateMusic.ts

```
@@ -0,0 +1,247 @@
+      normalization_strategy: "loudness",
+    };
+
+    if (options.tempo !== undefined) {
+      inputPayload.bpm = options.tempo;
+    }
+    if (options.referenceAudio) {
+      const ref = await this.resolveBuffer(options.referenceAudio);
+      inputPayload.input_audio = `data:audio/${this.detectAudioType(ref)};base64,${ref.toString("base64")}`;
+    }
+
+    let prediction;
```

Biome (2.4.15)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)